fenix/app/metrics.yaml

```diff
@@ -10828,12 +10777,170 @@ shopping.settings:
       - https://bugzilla.mozilla.org/show_bug.cgi?id=1843508
     data_reviews:
       - https://github.com/mozilla-mobile/firefox-android/pull/3619
+      - https://github.com/mozilla-mobile/firefox-android/pull/4039
     data_sensitivity:
       - interaction
     notification_emails:
       - android-probes@mozilla.com
-      - cgordon@mozilla.com
-    expires: 131
+    expires: never
     metadata:
       tags:
         - Shopping
+fx_suggest:
+  ping_type:
+    type: string
+    description: >
+      The ping's type. In other situations might be designated by an event's
+      name or an interaction field. E.g. "fxsuggest-impression",
+      "fxsuggest-block", "fxsuggest-click".
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1857092
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3958#issuecomment-1758607927
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - lina@mozilla.com
+      - ttran@mozilla.com
+      - najiang@mozilla.com
+    expires: never
+    send_in_pings:
+      - fx-suggest
+  source:
+    type: string
+    description: |
+      The source of the interaction. E.g. "urlbar".
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1857092
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3958#issuecomment-1758607927
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - lina@mozilla.com
+      - ttran@mozilla.com
+      - najiang@mozilla.com
+    expires: never
+    send_in_pings:
+      - fx-suggest
+  match_type:
```

Comment:
I wonder, should we omit `match_type` for now, or keep it for future compatibility? Mobile doesn't have `best-match`.

Reply:
makes sense